webui/react/src/utils/data.ts

```diff
@@ -149,3 +149,31 @@ export const isEqual = (a: unknown, b: unknown): boolean => {
   if (a === b) return true;
   return JSON.stringify(a) === JSON.stringify(b);
 };
+
+// export const getSubObjectCopy = (obj: RawJson, accessors: string[]): any => {
+//   return accessors
+//     .reduce((acc, cur) => acc === undefined ? acc : acc[cur], obj);
+// };
+
+// returns the sub object access by accessors (by reference).
+export const getSubObject = (obj: RawJson, accessors: string[]): any => {
```

Comment:
**question:** doesn't `getPath` attempt to do the same thing with the exception of the different accessors/path format?

Reply:
huh I forgot we have that let me look

Reply:
using the reducer approach solved this. I added a test.